apispec/core.py

```diff
@@ -182,6 +182,13 @@ def add_path(self, path=None, operations=None, **kwargs):
     def definition(self, name, properties=None, enum=None, **kwargs):
         """Add a new definition to the spec.
 
+        If you are using the Marshmallow extension, you can pass fields'metadatas in the
```

Comment:
Not sure if this should go here; this should probably go in the documentation for the marshmallow extension.